### PR TITLE
Add missing endpoints

### PR DESCRIPTION
**Pull request by marcinwadon**

### CODE REVIEW

File: modules/core/src/main/scala/org/tessellation/http/routes/DagRoutes.scala [hunk 1]
```diff
@@ -7,7 +7,8 @@ import cats.syntax.functor._
 import org.tessellation.dag.domain.block.L1Output
 import org.tessellation.domain.cell.{L0Cell, L0CellInput}
 import org.tessellation.domain.dag.DAGService
-import org.tessellation.ext.http4s.vars.{AddressVar, SnapshotOrdinalVar}
+import org.tessellation.ext.http4s.AddressVar
+import org.tessellation.ext.http4s.vars.SnapshotOrdinalVar
```
Comment (TheMMaciek) [inline modules/core/src/main/scala/org/tessellation/http/routes/DagRoutes.scala:11]: If we are moving `AddressVar` to it's own file shouldn't we delete the `vars` object and move `SnapshotOrdinalVar` to it's own file as well?

Comment (marcinwadon) [reply modules/core/src/main/scala/org/tessellation/http/routes/DagRoutes.scala]: I can do that :+1:

**Review by TheMMaciek — APPROVED** [review]

File: modules/core/src/main/scala/org/tessellation/http/routes/DagRoutes.scala [hunk 2]
```diff
@@ -7,7 +7,8 @@ import cats.syntax.functor._
 import org.tessellation.dag.domain.block.L1Output
 import org.tessellation.domain.cell.{L0Cell, L0CellInput}
 import org.tessellation.domain.dag.DAGService
-import org.tessellation.ext.http4s.vars.{AddressVar, SnapshotOrdinalVar}
+import org.tessellation.ext.http4s.AddressVar
+import org.tessellation.ext.http4s.SnapshotOrdinalVar
```
Comment (TheMMaciek) [inline modules/core/src/main/scala/org/tessellation/http/routes/DagRoutes.scala:11]: Let's hope that this will not fail the CI build ;)

Comment (TheMMaciek) [reply modules/core/src/main/scala/org/tessellation/http/routes/DagRoutes.scala:11]: I mean formatting :)

Comment (marcinwadon) [reply modules/core/src/main/scala/org/tessellation/http/routes/DagRoutes.scala:11]: oh 🙈

**Review by TheMMaciek — APPROVED** [review]

**Pull request merged**